src/components/cluster/detail/WorkerNodesAWS.js

```diff
@@ -15,6 +16,23 @@ function WorkerNodesAWS({
   workerNodesDesired,
   workerNodesRunning,
 }) {
+  const instanceTypeText = instanceType
+    ? `${instanceType.cpu_cores} CPUs, ${instanceType.memory_size_gb.toFixed(
```

Comment:
Can you please move this to a separate function? It's hard to read

Reply:
Sorry, I don't find it hard to read, I guess if I make a function of it it will look quite similar... maybe it's because this `0` in its own line?
Could you please provide an example of how you would like it to be so I can understand it better?

Reply:
```js
const getInstanceTypeLabel = (instanceType) => {
  const cpuCores = 0;
  const memorySize = 0;

  if (instanceType) {
    cpuCores = instanceType.cpu_cores;
    memorySize = instanceType.memory_size_gb.toFixed(0);
  }

  const label = `${cpuCores} CPUs, ${memorySize} GB RAM`;

  return label;
}

const instanceTypeLabel = getInstanceTypelabel(instanceType);
```

Reply:
Ok, then we don't agree here... 😢 I find this too verbose 😛 

@oponder help us please!

Reply:
I think prettier is making it hard here, because the line indeed got long and then it splits it in a nasty way.

Like this I can follow it fine and get the intent quickly:

```
  const instanceTypeText = instanceType
    ? `${instanceType.cpu_cores} CPUs, ${instanceType.memory_size_gb.toFixed(0)} GB RAM`
    : '0 CPUs, 0 GB RAM';
```

Not sure what the best approach here is. Adrian's is more verbose for sure, but I do find it quicker to get the intent when compared to:

```
  const instanceTypeText = instanceType
    ? `${instanceType.cpu_cores} CPUs, ${instanceType.memory_size_gb.toFixed(
        0
      )} GB RAM`
    : '0 CPUs, 0 GB RAM';
```

Just because of the wacky line break, makes me spend a few more cpu cycles trying to figure out what is going on there.

Reply:
And what if we use `// prettier-ignore` so the line isn't splitted?

Reply:
I leave it up to you @ograu! 

I would also be ok with `// prettier-ignore` . Just tested that out and it leaves it alone.

```
  // prettier-ignore
  const instanceTypeText = instanceType
    ? `${instanceType.cpu_cores} CPUs, ${instanceType.memory_size_gb.toFixed(0)} GB RAM`
    : '0 CPUs, 0 GB RAM';
```

And then maybe we have a discussion later about increasing the prettier line length. But that has it's own issues

Reply:
are you ok with this @axbarsan ?

```javascript
  // prettier-ignore
  const instanceTypeText = instanceType
    ? `${instanceType.cpu_cores} CPUs, ${instanceType.memory_size_gb.toFixed(0)} GB RAM`
    : '0 CPUs, 0 GB RAM';
```

Reply:
For me readibility is way more important, compared to not being verbose.

But you can go ahead and do it your way 👍

Reply:
> For me readibility is way more important, compared to not being verbose.

Readability is paramount for me as well. I think it's just that we have different ideas of what's readable and what's not. And that's natural, we just started working together. That's why I asked for an example, because I want to understand you better so maybe I can make my code more readable to you. Hope you can appreciate my effort in trying to understand your point.

> But you can go ahead and do it your way 👍

And yeah, "my way", if there is no consensus on how to do/write something I think it makes sense to leave it up to the author of the PR.

BTW I suggest that if you have strong opinions and clear ideas about readability, interpolations, etc. please feel free to open a PR in our coding guidelines repo and let's keep this kind of discussions there. This way we could agree in things and there will be no need to have discussions this long in PRs.

Meanwhile, and especially when there is no consensus, I propose let's be a little bit flexible with the work of our peers.